### PR TITLE
Repository.Hook: Add missing fields Type, Name, TestURL, PingURL and LastResponse

### PR DESCRIPTION
**Pull request by andygrunwald**
**What type of PR is this?**

feature

**What this PR does / why we need it**:

Adds support for missing fields in the Repository.Hook structure.

Here is an example of a full response of a public repo, where I am the owner + when I call ListHooks

```
{
    "type": "Repository",
    "id": <ID> (int),
    "name": "web",
    "active": true,
    "events": [
      "create",
      "delete",
      "issue_comment",
      "member",
      "public",
      "pull_request",
      "push",
      "repository"
    ],
    "config": {
      "url": "https://notify.travis-ci.org",
      "insecure_ssl": "0",
      "content_type": "form"
    },
    "updated_at": "2019-01-15T15:40:01Z",
    "created_at": "2019-01-15T15:40:01Z",
    "url": "https://api.github.com/repos/andygrunwald/go-jira/hooks/<ID>",
    "test_url": "https://api.github.com/repos/andygrunwald/go-jira/hooks/<ID>/test",
    "ping_url": "https://api.github.com/repos/andygrunwald/go-jira/hooks/<ID>/pings",
    "last_response": {
      "code": 204,
      "status": "active",
      "message": "OK"
    }
  }
```

**Special notes for your reviewer**:

None

**Does this PR introduce a user-facing change?**:

None

**Additional documentation e.g., usage docs, etc.**:

- [List repository webhooks @ GitHub REST API docs](https://docs.github.com/en/rest/reference/repos#list-repository-webhooks)

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @andygrunwald !
LGTM.

Awaiting second LGTM before merging.

**Review by wesleimp — APPROVED** [review]
LGTM 👌🏼

**Comment by gmlewis** [comment]
Thank you, @wesleimp !
Merging.

**Pull request merged**
